app/assets/javascripts/quby/answers/validation.js

```diff
@@ -189,6 +208,48 @@
     return !failed;
   };
 
+  function parsePartialDate(inputs) {
+    var value = function(placeholder) {
+      var val = inputs.filter("[placeholder=" + placeholder + "]").first().val();
+      if(val === undefined || val == "")
```

Comment:
Dat krijg je in de praktijk niet, want in valid_date wordt voordat parsePartialDate wordt aangeroepen hetvolgende gedaan:

````javascript
fieldsEmpty = numberOfEmptyDateFields(inputs);
if(fieldsEmpty > 0 && fieldsEmpty < $(inputs).size()) {
  throw "invalidDate";
}
````